# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../procps/procps-top-DL_LIB.patch
# Copyright (C) 2014 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Remove ${DL_LIB} from the LDFLAGS

strangely the Makefile for `top` includes this:
--------------------------------------------------------------------------
DL_LIB = none required
--------------------------------------------------------------------------

which leads to an error when linking:
--------------------------------------------------------------------------
  CCLD   top
i686-unknown-linux-musl-gcc: error: none: No such file or directory
i686-unknown-linux-musl-gcc: error: required: No such file or directory
--------------------------------------------------------------------------

--- a/top/Makefile.am
+++ b/top/Makefile.am
@@ -4,7 +4,6 @@
 	-DLOCALEDIR=\"$(localedir)\"
 
 AM_LDFLAGS = ../proc/libprocps.la
-AM_LDFLAGS += $(DL_LIB)
 
 if WITH_NCURSES
 usrbin_exec_PROGRAMS = \
